sale_quotation_capable_vendor_rfq/models/product_product.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_matching_vendor_pricelists(self, quantity, output_time=0):
+        """
+        Returns a recordset of product.supplierinfo records that match the criteria.
+
+        :param quantity: Quantity to match
+        :param output_time: Integer of days to match
+        :return: Recordset of product.supplierinfo
+        """
+        self.ensure_one()
+
+        # get all vendors that match the criteria
+        supplierinfo = self.seller_ids.filtered(
+            lambda s: (not output_time or s.delay >= output_time)
```

Comment:
Let's assume `output_time == 10`. This means that we need to find all vendors that can deliver faster than 10 days.
Shouldn't it be `s.delay <= output_time`?